.github/ISSUE_TEMPLATE/kubernetes_bump.md

```diff
@@ -30,10 +30,16 @@ changes should be cherry-picked to all release series that will support the new
   * Prior art: #7156
   * bump `InitWithKubernetesVersion` in `clusterctl_upgrade_test.go`
 * [ ] Job configurations:
-  * For all releases which will support the new Kubernetes version:
-    * Update `INIT_WITH_KUBERNETES_VERSION`.
-    * Add new periodic upgrade jobs .
+  * Note: The jobs are adjusted to have test coverage according to our [support policy](https://cluster-api.sigs.k8s.io/reference/versions.html#supported-kubernetes-versions).
```

Comment:
should this be:
```suggestion
  Ensure the jobs are adjusted to provide test coverage according to our [support policy](https://cluster-api.sigs.k8s.io/reference/versions.html#supported-kubernetes-versions):
```
So it is slightly clear what/why the task is about.

Reply:
Good idea. I moved it also up one level